Sources/SwiftParser/Directives.swift

```diff
@@ -103,32 +109,62 @@ extension Parser {
     do {
       var firstIteration = true
       var loopProgress = LoopProgressCondition()
-      while let poundIfHandle = firstIteration ? self.canRecoverTo(.poundIfKeyword) : self.canRecoverTo(anyIn: IfConfigContinuationClauseStartKeyword.self)?.handle,
-        loopProgress.evaluate(self.currentToken)
-      {
-        var (unexpectedBeforePoundIf, poundIf) = self.eat(poundIfHandle)
-        firstIteration = false
-        // Parse the condition.
+      LOOP: while let (match, handle) = self.canRecoverTo(anyIn: PoundIfDirectiveKeywords.self), loopProgress.evaluate(self.currentToken) {
+        var unexpectedBeforePound: RawUnexpectedNodesSyntax?
+        var pound: RawTokenSyntax
```

Comment:
What do you think about naming this `poundKeyword` instead of `pound`. Because in almost all cases, it won’t be a `#` by itself. Same for `unexpectedBeforePound` -> `unexpectedBeforePoundKeyword`.